man/git-extras.md

```diff
@@ -1,100 +0,0 @@
-git-extras(1) -- Awesome GIT utilities
```

Comment:
Why remove this file?

Reply:
I didn't figure out how `make docs` work so I messed with it a bit.
I believe I've fixed it now.